packages/edit-post/src/components/layout/index.js

```diff
@@ -414,6 +415,8 @@ function Layout( {
 			const { isZoomOut } = unlock( select( blockEditorStore ) );
 			const { getEditorMode, getRenderingMode } = select( editorStore );
 			const isRenderingPostOnly = getRenderingMode() === 'post-only';
+			const isNotDesignPostType =
+				! DESIGN_POST_TYPES.includes( currentPostType );
```

Comment:
Unrelated micro optimization here so that the couple of places this was evaluated can share the result (and read a little better I think).